api/python/quilt3/main.py

```diff
@@ -99,9 +101,10 @@ def _launch_local_s3proxy():
 https://quiltdata.com for more information.
 """
 
-def cmd_catalog(s3_url=None, detailed_help=False):
+def cmd_catalog(navigation_target=None, detailed_help=False):
     """
-    Run the Quilt catalog locally.
+    Run the Quilt catalog locally. If navigation_targets starts with 's3://', open file view. Otherwise assume it
+    refers to a package, following the pattern: BUCKET:USER/PKG
```

Comment:
I'd suggest an explicit scheme for package links just :`quilt://BUCKET:USER/PKG[/SUB_PACKAGE][@VERSION]` (no no need to support sub-packages and versions in this PR. i'm just demonstrating why we want to be more formal with "logical addressing" and have a scheme for it.